KnowYourEnemyMutagen/Program.cs

```diff
@@ -39,315 +29,219 @@ public static int Main(string[] args)
                 });
         }
 
-
-        public static void RunPatch(SynthesisState<ISkyrimMod, ISkyrimModGetter> state)
+        private static float AdjustDamageMod(float magnitude, float scale)
```

Comment:
The reason why camelCase and snake_case consistency is all over the place is because I was keeping an eye on the original patcher, it was easier to compare when I used the same names on my own functions.